cookbook/README.md

```diff
@@ -301,6 +317,33 @@ POST /transactions/:conn_id/set-endorser-info?endorser_did=8WzWX4G3Rti6tVSX3Atcv
 }
 ```
 
+### 4. Set public DID
+To make the wallet aware which DID should be used, the DID on the ledger must be set to public.
+
+* :did must be replaced with the DID registered on the sandbox ledger
+* :conn_id must be replaced with the connection_id, as in the previous step
```

Comment:
I think this formatting is not used elsewhere in this document, to make it uniform we should probably add it to the other requests as well.

Reply:
Resolution: Added formatted params to every request
> Params
> `did` must be replaced with the DID registered on the sandbox ledger
> `conn_id` must be replaced with the connection_id, as in the previous step